scripts/subql-diff/README.md

```diff
@@ -0,0 +1,31 @@
+## Compare Subquery Data Diff
+Compares two copies of subquery data, and find out extra/missing/diff records.
+use `--full` to show full difference, or `--no-full` to only show different ids
+
+### run with example data
+first extract the data, which contains some valid karura receipts/logs, and another copy of data that missed some record.
+```
+tar -xvf data.tgz
+```
+
+check for receipts diff
+```
+yarn diff                                \
```

Comment:
better to have this published to npm and use `npx` to run so people don't need to clone & setup this repo